libs/jsonschema/schema.go

```diff
@@ -76,6 +76,10 @@ type Schema struct {
 	// Title of the object, rendered as inline documentation in the IDE.
 	// https://json-schema.org/understanding-json-schema/reference/annotations
 	Title string `json:"title,omitempty"`
+
+	// Examples of the value for properties in the schema.
+	// https://json-schema.org/understanding-json-schema/reference/annotations
+	Examples []any `json:"examples,omitempty"`
```

Comment:
Do you know if the schema check we perform also validates that entries here match the containing schema?